xfbin/structure/br/br_nucc.py

```diff
@@ -95,58 +95,66 @@ def init_data(self, br: BinaryReader):
             self.brNut = None
 
         # Skip the nut size
-        br.seek(self.nutSize, Whence.CUR)
+        #br.seek(self.nutSize, Whence.CUR)
 
     def __br_write__(self, br: 'BinaryReader', chunkIndexDict: IterativeDict):
         # TODO: Actual NUT writing
```

Comment:
This TODO comment (and the following lines) should be removed since the task is done.